.circleci/config.yml

```diff
@@ -224,6 +224,9 @@ jobs:
       - run:
           name: Check missing file headers
           command: node ./scripts/tasks/check-license-headers.js
+      - run:
+          name: Check package.json integrity
+          command: node ./scripts/tasks/check-and-rewrite-package-json.js --test
```

Comment:
This should run as part of Husky, so it _shouldn't_ get out of sync, but this is just a failsafe.